src/components/Sidebar/Sidebar.jsx

```diff
@@ -20,11 +41,46 @@ export const Sidebar = ({ children, className, footer, notifications, isFixed =
                 className,
             )}
         >
-            <div className={clsx("p-2 text-center xl:text-left", notifications ? null : "invisible")}>
-                <Counter className="text-sm">{notifications}</Counter>
+            <div className="flex p-2 sm:justify-center xl:justify-between">
+                <div
+                    className={clsx(
+                        "cursor-pointer sm:text-center",
+                        notifications?.announcements?.count ? null : "hidden",
```

Comment:
`announcements` and `notices` are too specific for UI Kit. They both are seller app features. Let's call this left drawer and right drawer.

@nemanjakrstic Name suggestions welcome